database/migrations/2024_03_11_132400_seed_options_list_siting_strategy.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+use App\Models\V2\Forms\FormOptionList;
+use App\Models\V2\Forms\FormOptionListOption;
+use App\Models\V2\I18n\I18nItem;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Str;
+
+class SeedOptionsListSitingStrategy extends Migration
+{
+    /**
+     * This migration seeds the default options
+     *
+     * @return void
+     */
+    public function up()
+    {
+        $collections = [
+            'siting-strategy-collection' => ['Concentred','Distributed', 'Hybrid'],
+        ];
+
+        foreach ($collections as $key => $items) {
+            $formOptionList = FormOptionList::create(['key' => $key]);
+
+            foreach ($items as $item) {
+                $options = FormOptionListOption::create([
+                    'form_option_list_id' => $formOptionList->id,
+                    'label' => $item,
+                    'slug' => Str::slug($item),
+                ]);
+            }
+        }
+
+        foreach (FormOptionListOption::all() as $option) {
+            $option->label_id = $this->generateIfMissingI18nItem($option, 'label');
+            $option->save();
+        }
+    }
+
+    private function generateIfMissingI18nItem(Model $target, string $property): ?int
+    {
+        $value = trim(data_get($target, $property, false));
+        $short = strlen($value) <= 256;
+        if ($value && data_get($target, $property . '_id', true)) {
+            $i18nItem = I18nItem::create([
+                'type' => $short ? 'short' : 'long',
+                'status' => I18nItem::STATUS_DRAFT,
+                'short_value' => $short ? $value : null,
+                'long_value' => $short ? null : $value,
+            ]);
+
+            return $i18nItem->id;
+        }
+
+        return data_get($target, $property . '_id');
+    }
```

Comment:
to avoid using every time a new function called `generateIfMissingI18nItem` which is in multiple places, can you try to use the `I18nHelper`?